src/documents/views.py

```diff
@@ -679,13 +679,16 @@ def get(self, request, format=None):
         if settings.ENABLE_UPDATE_CHECK:
             try:
                 with urllib.request.urlopen(
-                    "https://raw.githubusercontent.com/paperless-ngx"
-                    + "/paperless-ngx/main/src/paperless/version.py",
+                    "https://api.github.com/repos/"
+                    + "paperless-ngx/paperless-ngx/releases/latest",
                 ) as response:
                     remote = response.read().decode("utf-8")
-                match = re.search(r"(\d+, \d+, \d+)", remote)
+                match = re.search(
+                    r'"tag_name":\s*"ngx-(\d+\.\d+\.\d+)"',
```

Comment:
It looks like the Github API comes back with JSON, so we could now `json.load` that, access tag_name and then search out the version from there.

Reply:
Can you tell I dont do a lot go python. Cool thanks! (updated)